code/ui/.storybook/main.ts

```diff
@@ -67,6 +67,11 @@ const config: StorybookConfig = {
       configType === 'PRODUCTION' ? pluginTurbosnap({ rootDir: viteConfig.root || '' }) : [],
     ],
     optimizeDeps: { ...viteConfig.optimizeDeps, force: true },
+    build: {
+      ...viteConfig.build,
+      // disable sourcemaps in CI to not run out of memory
+      sourcemap: process.env.CI !== 'true',
```

Comment:
I verified that CircleCI sets it to `"true"` and not just "1" or something.  https://circleci.com/docs/variables/#built-in-environment-variables